worlds/minecraft/Locations.py

```diff
@@ -124,6 +124,15 @@ def __init__(self, player: int, name: str, address: typing.Optional[int], parent
     "Sound of Music": AdvData(42105, 'Overworld'),
     "Star Trader": AdvData(42106, 'Village'),
 
+    # 1.19 advancements
+    "Birthday Song": AdvData(42107, 'Overworld'),
```

Comment:
This advancement and the other Allay advancement should be in the Pillager Outpost region.